samples/msal-angular-v2-samples/angular10-sample-app/test/home.spec.ts

```diff
@@ -35,14 +41,25 @@ describe('/ (Home Page)', () => {
         port = global.__PORT__;
 
         const labApiParams: LabApiQueryParams = {
-            azureEnvironment: AzureEnvironments.PPE,
-            appType: AppTypes.CLOUD
+            azureEnvironment: AzureEnvironments.CLOUD,
+            appType: AppTypes.CLOUD,
+            userType: UserTypes.CLOUD,
+            appPlatform: AppPlatforms.SPA,
         };
 
         const labClient = new LabClient();
         const envResponse = await labClient.getVarsByCloudEnvironment(labApiParams);
 
         [username, accountPwd] = await setupCredentials(envResponse[0], labClient);
```

Comment:
There are no credentials being used here since javascript apps are either user agent or public client applications.

Reply:
Maybe a naming confusion -these are username and password for the test account used in automation

Reply:
Let us rename them later in a separate PR - `setupTestAccCredentials()`?

Reply:
Sounds good, I made a note of that